keyboards/anavi/knob1/keymaps/default/keymap.c

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Leon Anavi <leon@anavi.org>
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+#include <stdio.h>
+
+enum layer_names {
+    _BASE
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [_BASE] = LAYOUT_k1(
+        KC_MUTE
+    )
+};
+
+#ifdef OLED_ENABLE
+
+bool oled_task_user(void) {
+  // Host Keyboard Layer Status
+  oled_write_ln_P(PSTR("ANAVI Knob 1"), false);
+  oled_write_ln_P(PSTR("Keymap: Default"), false);
+
+  // Host Keyboard LED Status
+  led_t led_state = host_keyboard_led_state();
+  oled_write_P(PSTR("Num Lock: "), false);
+  oled_write_ln_P(led_state.num_lock ? PSTR("On") : PSTR("Off"), false);
+  oled_write_P(PSTR("Caps Lock: "), false);
+  oled_write_ln_P(led_state.caps_lock ? PSTR("On") : PSTR("Off"), false);
+  oled_write_P(PSTR("Scroll Lock: "), false);
+  oled_write_ln_P(led_state.scroll_lock ? PSTR("On") : PSTR("Off"), false);
+#ifdef RGBLIGHT_ENABLE
+  static char rgbStatusLine1[26] = {0};
+  snprintf(rgbStatusLine1, sizeof(rgbStatusLine1), "RGB Mode: %d", rgblight_get_mode());
```

Comment:
https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

This will save a good chunk of space. (~1.5k)

Reply:
Thank you for the code review. May I keep it like this as there is plenty of space on the RP2040 for the needs of this simple mini keyboard?